File: internal/content/testdata/bench/typescript/nth-fibonacci-list.ts
function fibonacci_list(n: number): number[] {
    if (n === 0) return [];
    if (n === 1) return [0];
    const result: number[] = [0, 1];
    for (let i = 2; i < n; i++) {
        result.push(result[result.length - 1] + result[result.length - 2]);
    }
    return result;
}

File: internal/content/testdata/bench/rust/nth-fibonacci-list.rs
fn fibonacci_list(n: i64) -> Vec<i64> {
    if n == 0 {
        return vec![];
    }
    if n == 1 {
        return vec![0];
    }
    let mut result: Vec<i64> = vec![0, 1];
    for _ in 2..n {
        let len = result.len();
        result.push(result[len - 1] + result[len - 2]);
    }
    result
}
